cse-valid-adapter-app/src/main/resources/application.yml

```diff
@@ -1,36 +1,4 @@
 spring:
-  rabbitmq:
-    username: gridcapa
-    password: gridcapa
-    host: localhost
-    port: 5672
   cloud:
-    stream:
-      bindings:
-        consumeTask-in-0:
-          binder: rabbit
-          destination: cse-valid-run-task
-          group: cse-valid
-        task-status-update:
-          destination: task-manager-status-update
-      rabbit:
-        default:
-          consumer:
-            auto-bind-dlq: true
-            max-attempts: 1
     function:
       definition: consumeTask;consumeAutoTask
```

Comment:
Does the "consumeAutoTask" function really exist ? I can't find it in the code of this adapter

Reply:
You are right, there is no "consumeAutoTask" method in CSE Valid Adapter. I will remove the useless definition.